crypto/rand.go

```diff
@@ -21,12 +21,16 @@ func RandIntn(n int) (int, error) {
 
 // Perm returns, as a slice of n ints, a random permutation of the integers
 // [0,n).
-func Perm(n int) []int {
+func Perm(n int) ([]int, error) {
```

Comment:
I didn't like it at first either, but then I saw that the other rand functions also return errors.